CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+### Updated
+- Updated Plotly.js from version 2.30.0 to version 2.31.0. See the [plotly.js CHANGELOG](https://github.com/plotly/plotly.js/blob/master/CHANGELOG.md#2310----2024-04-10) for more information. These changes are reflected in the auto-generated `plotly.graph_objects` module. Notable changes include:
```

Comment:
Please bump plotly.js version to 2.31.1: https://github.com/plotly/plotly.js/releases/tag/v2.31.1
Thank you!

Reply:
Done